misc/experimental/rust/CMakeLists.txt

```diff
@@ -26,13 +26,12 @@ if (CELIX_RUST_EXPERIMENTAL)
     FetchContent_MakeAvailable(Corrosion)
 
     #Prepare a list of include paths needed to generating bindings for the Apache Celix C API
-    file(GENERATE
-            OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/include_paths.txt"
-            CONTENT "$<TARGET_PROPERTY:framework,INTERFACE_INCLUDE_DIRECTORIES>;$<TARGET_PROPERTY:utils,INTERFACE_INCLUDE_DIRECTORIES>"
-    )
+    #Note for now this includes framework, utils and shell_api maybe this should be separated in the future.
+    file(GENERATE OUTPUT
+        "${CMAKE_CURRENT_BINARY_DIR}/include_paths.txt" CONTENT
+        "$<TARGET_PROPERTY:framework,INTERFACE_INCLUDE_DIRECTORIES>;$<TARGET_PROPERTY:utils,INTERFACE_INCLUDE_DIRECTORIES>;$<TARGET_PROPERTY:shell_api,INTERFACE_INCLUDE_DIRECTORIES>;$<TARGET_PROPERTY:Celix::log_service_api,INTERFACE_INCLUDE_DIRECTORIES>")
```

Comment:
`INCLUDE_DIRECTORIES` alone is not enough, we shall also have `COMPILE_DEFINITIONS` and `COMPILE_OPTIONS`.